packages/table/src/table2.tsx

```diff
@@ -1624,4 +1608,19 @@ export class Table2 extends AbstractComponent<Table2Props, TableState, TableSnap
     private handleRowResizeGuide = (horizontalGuides: number[]) => {
         this.setState({ horizontalGuides });
     };
+
+    private getHeaderDimensions = (): TableHeaderDimensions => {
+        return {
+            columnHeaderHeight: this.getColumnHeaderHeight(),
+            rowHeaderWidth: this.getRowHeaderWidth(),
+        };
+    };
+
+    private getColumnHeaderHeight = (): number => {
```

Comment:
my preference would be to merge all no-op refactors like this in a setup PR so we can really be reviewing a minimal diff here

Reply:
I actually tried putting up a PR for noops but they are so minimal they should hardly be a distractor. For instance, `getColumnHeaderHeight` here clearly just encapsulates the implementation from elsewhere in the class.